tomviz/RotateAlignWidget.cxx

```diff
@@ -757,6 +777,11 @@ void RotateAlignWidget::updateControls()
 
   this->Internals->axesActor->SetXAxisRange(xAxisRange);
   this->Internals->axesActor->SetYAxisRange(yAxisRange);
+
+  // It would be nice if we could only write the settings when the
+  // widget is accepted, but I don't immediately see an easy way
+  // to do that.
```

Comment:
Yep, would be nice `EditOperatorDialog` has a `editEnded` signal that we could probably connect up to the `EditOperatorWidget`, but I don't think its worth it.